apps/explorer/src/pages/AddressPage/AddressTransactions.tsx

```diff
@@ -27,32 +25,22 @@ const AddressTransactions = ({ addressStr }: AddressTransactionsProps) => {
   const isAppVisible = usePageVisibility()
   const pageNumber = usePageNumber()
   const isContract = useIsContract(addressStr)
-  const lastKnownMempoolTxs = useRef<MempoolTransaction[]>([])
 
   const { data: txNumber } = useFetchAddressTransactionsCount(addressStr)
 
-  const {
-    data: txList,
-    isLoading: txListLoading,
-    refetch: refetchTxList
-  } = useQuery({
+  const refetchInterval = isAppVisible && pageNumber === 1 ? 10000 : undefined
+
+  const { data: txList, isLoading: txListLoading } = useQuery({
     ...queries.address.transactions.confirmed(addressStr, pageNumber, numberOfTxsPerPage),
-    enabled: !!addressStr,
+    refetchInterval,
     placeholderData: keepPreviousData
   })
 
   const { data: addressMempoolTransactions = [] } = useQuery({
     ...queries.address.transactions.mempool(addressStr),
-    refetchInterval: isAppVisible && pageNumber === 1 ? 10000 : undefined
+    refetchInterval
```

Comment:
We need to poll both queries to avoid cases where a tx goes in and out of the mempool during the interval time (10s)